packages/config/config/devices/0x027a/zen23_3.0.json

```diff
@@ -1,6 +1,7 @@
 // Zooz ZEN23
 // Z-Wave Plus On/Off Toggle Switch V3
 {
+	"revision": 10,
```

Comment:
You can remove this - we've opted not to import OZW files, so this is no longer necessary.